src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudApp.scala

```diff
@@ -1,25 +1,18 @@
 package org.broadinstitute.dsde.firecloud
 
 import scala.concurrent.duration._
-
 import akka.actor.{ActorSystem, Props}
-import org.slf4j.LoggerFactory
-
+import com.typesafe.scalalogging.slf4j.LazyLogging
 import org.broadinstitute.dsde.vault.common.util.ServerInitializer
 
-object FireCloudApp {
+object FireCloudApp extends LazyLogging {
 
   // we need an ActorSystem to host our application in
   val system = ActorSystem("FireCloud-Orchestration-API")
   val timeoutDuration = FiniteDuration(FireCloudConfig.HttpConfig.timeoutSeconds, SECONDS)
 
-  lazy val log = LoggerFactory.getLogger(getClass)
-
-  // create and start our service actor
-  val service = system.actorOf(Props[FireCloudServiceActor], "FireCloudService")
-
```

Comment:
this is embarrassing. We were starting FireCloudServiceActor twice - once in the above line, and once inside startWebServiceActors below. This is what caused the ElasticSearchDAO.initIndex call to be logged twice - it actually did happen twice.